scripts/fetch-from-github/fetch.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const path = require('path');
+
+// Function to load Octokit once
+let octokitInstance = null;
+
+async function loadOctokit() {
+  if (!octokitInstance) {
+    const { Octokit } = await import('@octokit/rest');
+    octokitInstance = process.env.VBOT_GITHUB_API_TOKEN
```

Comment:
Should this be REDPANDA_GITHUB_TOKEN to match other usages in docs-extensions-and-macros?

Reply:
this doesn't get executed in the same environment so the name isn't so important. We're just getting it from AWS, which also has a different name 😄